src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,109 @@
+.sidebar {
+  position: fixed;
```

Comment:
Let's let the app that imports this component worry about its position.  We don't want to enforce anything here that can hurt this component's flexibility.